custom_components/tahoma/sensor.py

```diff
@@ -72,9 +74,20 @@ def unit_of_measurement(self):
         if self.tahoma_device.uiclass == "LightSensor":
             return "lx"
 
-        if self.tahoma_device.uiclass == "ElectricitySensor":
+        if (
+            self.tahoma_device.uiclass == "ElectricitySensor"
```

Comment:
Is `ElectricitySensor` necessary here? I like your approach even more, since it can be more modular.